Lib/test/libregrtest/setup.py

```diff
@@ -14,17 +14,29 @@
 
 
 def setup_tests(ns):
-    # Display the Python traceback on fatal errors (e.g. segfault)
-    faulthandler.enable(all_threads=True)
-
-    # Display the Python traceback on SIGALRM or SIGUSR1 signal
-    signals = []
-    if hasattr(signal, 'SIGALRM'):
-        signals.append(signal.SIGALRM)
-    if hasattr(signal, 'SIGUSR1'):
-        signals.append(signal.SIGUSR1)
-    for signum in signals:
-        faulthandler.register(signum, chain=True)
+    try:
+        stderr_fd = sys.__stderr__.fileno()
+    except (ValueError, AttributeError):
+        # In IDLE, sys.stderr.fileno() raises io.UnsupportedOperation: the
```

Comment:
If you use `sys.__stderr__.fileno()` this comment looks outdated.

Reply:
Right, removed.